source/core/utilities.cc

```diff
@@ -94,7 +94,113 @@ make_table_tensors_scalars(
   return table;
 }
 
+void
+fill_table_from_file(TableHandler &    table,
+                     std::string       file,
+                     const std::string delimiter)
+{
+  table.clear();
+  std::ifstream myfile(file);
+  // open the file
+  if (myfile.is_open())
+    {
+      std::string              line;
+      std::vector<std::string> vector_of_column_names;
+      std::vector<double>      line_of_data;
+      unsigned int             i = 0;
```

Comment:
Can you use another variable than "i" here. Because otherwise you have two i variables in the same functions. i understand that they are well defined because of {} encapsulation, but it really hinders readability.